tools/python/util/mobile_helpers/test/test_usability_checker.py

```diff
@@ -57,12 +67,21 @@ def test_scan_model(self):
 
             # print(log_capture)
             log_capture.check_present(
-                ("default", "INFO", "0 partitions with a total of 0/76 nodes can be handled by the NNAPI EP."),
-                ("default", "INFO", "72 nodes are in subgraphs, which are currently not handled."),
-                ("default", "INFO", "Unsupported ops: ai.onnx:Scan"),
+                ("default", "INFO", "4 partitions with a total of 72/76 nodes can be handled by the NNAPI EP."),
```

Comment:
nit: the comment describing the test seems out of date now: "All the real operators are in subgraphs, so we don't use NNAPI/CoreML currently."

Reply:
Updated